server/src/main/java/org/opensearch/action/search/SearchScrollAsyncAction.java

```diff
@@ -299,6 +299,7 @@ protected final void sendResponse(
                     successfulOps.get(),
                     0,
                     buildTookInMillis(),
+                    SearchResponse.PhaseTook.EMPTY,
```

Comment:
Better to pass it null like the rest of the logic. Otherwise you need all those -1 logic to determine whether to populate or not.

Reply:
Deleted all `SearchResponse.PhaseTook.EMPTY`